libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/processes-detail-item/dh-processes-detail-item.component.html

```diff
@@ -45,4 +48,7 @@
     <!--   class='open-in-new-icon'-->
     <!-- ></watt-icon>-->
   </div>
+  <div class="child-cell error-row" *ngIf="detail.errors.length > 0">
+    <dh-process-error-list [errors]="detail.errors"></dh-process-error-list>
```

Comment:
suggestion: instead of creating a custom component, consider using ["watt-validation-message"](https://61765fc47451ff003afe62ff-diqadctlgg.chromatic.com/?path=/docs/components-validation-message--validation-message) component, which was created specifically for that.

Reply:
Thanks, I looked for error messages but couldn't find them :)